python/chronos/src/bigdl/chronos/detector/anomaly/dbscan_detector.py

```diff
@@ -57,16 +57,23 @@ def check_data(self, arr):
         if len(arr.shape) > 1:
             raise ValueError("Only univariate time series is supported")
 
-    def fit(self, y):
+    def fit(self, y, use_sklearnex=True, algorithm_list=['DBSCAN']):
         """
         Fit the model
 
         :param y: the input time series. y must be 1-D numpy array.
+        :param use_sklearnex: bool, Whether to enable scikit-learn-intelex,
+               This default value is True.
```

Comment:
\+ If `scikit-learn-intelex` is not installed, `DBScanDetector` will fallback to use stack sklearn